courses-app/src/app/shared/shared.module.ts

```diff
@@ -1,30 +1,51 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import * as Components from './components/index';
+import * as Directives from './directives/index';
 import * as Pipes from './pipes/index';
+import * as Validators from './validators/index';
 import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
 
 import { FaIconLibrary } from '@fortawesome/angular-fontawesome';
 import { fas } from '@fortawesome/free-solid-svg-icons';
 import { far } from '@fortawesome/free-regular-svg-icons';
 import { fab } from '@fortawesome/free-brands-svg-icons';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { validateEmail } from './validators/index';
```

Comment:
what about to simplify and use 'path' declaration in tsconfig.json? because this part looks too overloaded

Reply:
can you please describe it more detailed, because i don't understand what to do in this case